relayer/client-tx.go

```diff
@@ -136,13 +137,31 @@ func (c *Chain) UpdateClients(dst *Chain) (err error) {
 }
 
 // UpgradesClients upgrades the client on src after dst chain has undergone an upgrade.
-func (c *Chain) UpgradeClients(dst *Chain) (err error) {
+func (c *Chain) UpgradeClients(dst *Chain) error {
```

Comment:
So this should be called after the chain hangs

Reply:
yea, I'm a little unsure about if the correct query height is being used